src/__experimental__/components/input-icon-toggle/input-icon-toggle.style.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import Colors from 'style/themes/base';
```

Comment:
This is not the correct way to extract themes from styled-components. The style rules are provided on the props of the component itself, and should be accessed on the callbacks passed into the template literals.

```
styled.div`
  background: ${(props) => props.theme.color};
`
```

Reply:
fixed